backend/Squirrel.Core/Squirrel.Core.DAL/Enums/CommentedEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Squirrel.Core.DAL.Enums;
+
+public enum CommentedEntity
+{
+    Project = 1,
```

Comment:
Remove 'Project', add 'Script'

Reply:
CommentedEntity should store names of entities for which we will be able to write comments, like 'Commits', 'Pullrequests' etc.
Maybe you thought about FileTypeEntity which should contain 'Script', 'Readme', weren't you?

Reply:
We'll add scripts later, but maybe you're right, we shouldn't add "Script" for now, but "Project" should still be removed

![image](https://github.com/BinaryStudioAcademy/bsa-2023-squirrel/assets/139565128/0da215bc-84f3-4179-a54d-c7c75572bbaf)

Reply:
maybe 'script' is for FileType enum?